python/ray/actor.py

```diff
@@ -494,58 +494,102 @@ def __init__(self, actor_id, actor_handle_id, actor_cursor, actor_counter,
         self.checkpoint_interval = checkpoint_interval
         self.class_name = class_name
 
+    @classmethod
```

Comment:
I'd like to get rid of this class (probably in a subsequent PR). This class basically says how to serialize/deserialize an `ActorHandle`, but that can probably be done by just registering a custom serializer for the `ActorHandle` class or something like that or maybe by setting the `__getstate__` and `__setstate__` methods of the class. Haven't completely thought through the details, but something like that may work.

Reply:
Agreed.